content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,237 @@
-reserved thanks to the *FIFO* strategy.
+As seen above, lot 000001 entered the stock first. :guilabel:`Create` and :guilabel:`Confirm` a
+sales order for six white shirts. All five shirts from lot 000001 and one shirt from lot 000002 will
+be selected to be sent to the customer.
 
-.. image:: removal/reserved-lots-FIFO.png
-   :align: center
-   :alt: two lots being reserved for sell with the FIFO strategy.
+Once the sales order is confirmed, the delivery order will be created and linked to the picking, and
+the oldest lot numbers will be reserved thanks to the FIFO strategy.
 
 LIFO (Last In, First Out)
 -------------------------
 
-With a *LIFO* strategy, that’s quite the opposite. In fact, the products that are brought the
-last move out the first. It is mostly used in case of products without a shelf life.
+With a :guilabel:`LIFO` strategy, the opposite is true. The products that are received last move out
+first. :guilabel:`LIFO` is mostly used for products without a shelf life.
 
-Even if our white shirts are clothes, we can say that they are timeless. So, let’s use them to
-test our *LIFO* strategy. Once again, open the product category via :menuselection:`Inventory
---> Configuration --> Product Categories` and change the removal strategy to *LIFO*.
+In this example, the white shirts will be used again to test the :guilabel:`LIFO` strategy. First,
+open the product category via :menuselection:`Inventory --> Configuration --> Product Categories`
+and change the :guilabel:`Force Removal Strategy` to :guilabel:`LIFO`.
 
 .. image:: removal/last-in-first-out.png
    :align: center
-   :alt: last in first out strategy set up as forced removal strategy.
+   :alt: Last in first out strategy set up as forced removal strategy.
 
-Now, create a sale order for 4 white shirts and check that the reserved products are from lots
-000003 and 000002.
-
-.. image:: removal/reserved-lots-LIFO.png
-   :align: center
-   :alt: two lots being reserved for sell with the LIFO strategy.
+Then, :guilabel:`Create` a sales order for four white shirts and :guilabel:`Confirm` it. Check that
+the reserved products are from lots 000003 and 000002 by looking at the :guilabel:`Detailed
+Operations` in the  :guilabel:`Sales Order`.
 
 .. important::
-   Don’t forget that the *LIFO* strategy is banned in many countries!
+   Don't forget that the :abbr:`LIFO (Last In, First Out)` strategy is banned in many countries!
 
-FEFO (First Expiry, First Out)
-------------------------------
+FEFO (First Expired, First Out)
+-------------------------------
 
-With the *FEFO* strategy, the way products are picked is not based on the reception date. In this
-particular case, they are dispatched according to their expiration date.
+With the :guilabel:`FEFO` removal strategy, the way products are picked is not based on the
+reception date. In this case, they are picked according to their expiration date.
```

Comment:
```suggestion
As explained earlier, the :abbr:`FEFO (First Expiry, First Out)` removal strategy differs from the :abbr:`FIFO (First In, First Out)` and :abbr:`LIFO (Last In, First Out)` strategies, because it targets products for removal based on **expiration dates** instead of their warehoues receipt dates.
```